File: week-04/day-02/GreenFox_org/Person.py
class Person:
    def __init__(self, name = "Jane Doe", age = 30, gender = "female"):
        self.name = name
        self.age = age
        self. gender = gender

    
    def introduce(self):
        print('Hi, I\'m {}, a {} years old {}'.format(self.name, self.age, self.gender))

    
    def get_goal(self):
        print("My goal is: Live for the moment!")

File: Csharp-course/week-03/day-05/GreenFox/GreenFox/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GreenFox
{
    class Person
    {
        protected string name;
        protected int age;
        protected string gender;

        public Person(string _name, int _age, string _gender)
        {
            this.name = _name;
            this.age = _age;
            this.gender = _gender;
        }

        public Person()
        {
            name = "Jane Doe";
            age = 30;
            gender = "female";
        }

        public virtual void Introduce()
        {
            Console.WriteLine("Hi, I'm {0}, a {1} year old {2}", this.name, this.age, this.gender);
        }

        public virtual void GetGoal()
        {
            Console.WriteLine("My goal is: Live for the moment!");
        }
    }
}
